frontend/public/components/storage-class.tsx

```diff
@@ -44,14 +44,20 @@ const StorageClassRow: React.SFC<StorageClassRowProps> = ({obj}) => {
 const StorageClassDetails: React.SFC<StorageClassDetailsProps> = ({obj}) => <React.Fragment>
   <div className="co-m-pane__body">
     <SectionHeading text="StorageClass Overview" />
-    <ResourceSummary resource={obj}>
-      <dt>Provisioner</dt>
-      <dd>{obj.provisioner || '-'}</dd>
-      <dt>Reclaim Policy</dt>
-      <dd>{obj.reclaimPolicy || '-'}</dd>
-      <dt>Default Class</dt>
-      <dd>{isDefaultClass(obj)}</dd>
-    </ResourceSummary>
+    <div className="row">
+      <div className="col-sm-6">
+        <ResourceSummary resource={obj} showNodeSelector={false} showPodSelector={false} >
```

Comment:
These props default to false and don't need to be specified

```suggestion
        <ResourceSummary resource={obj}>
```